src/pages/XmlUpload/components/DropZone/DropZone.jsx

```diff
@@ -20,27 +20,71 @@ const DropZone = ({ files, setFiles }) => {
     });
   };
 
+  const CONVERSION = 1e6; // = 1,000,000
```

Comment:
```suggestion
  const conversion= 1e6; // = 1,000,000
```

capital letters are for env variables only

Reply:
As I said, MACRO_CASE (also known as CONSTANT_CASE) is used when a variable is constant and is used solely to replace a magic value. It is a global convention. If you insist on it, I'll change it